var/spack/repos/builtin/packages/axom/package.py

```diff
@@ -50,6 +50,11 @@ class Axom(CachedCMakePackage, CudaPackage):
     version('0.3.0', tag='v0.3.0', submodules=True)
     version('0.2.9', tag='v0.2.9', submodules=True)
 
+    @property
+    def submodules(self):
```

Comment:
Does this mean all the `submodules=True` above can go away?

Reply:
Yes, I could modify that here if you like

Reply:
I'll make a separate PR for this